src/features/surveys/components/surveyForm/SurveyOptionsQuestion.tsx

```diff
@@ -0,0 +1,107 @@
+import { FC } from 'react';
+import {
+  Checkbox,
+  FormControl,
+  FormControlLabel,
+  FormGroup,
+  FormLabel,
+  MenuItem,
+  Radio,
+  RadioGroup,
+  Select,
+} from '@mui/material';
+import {
+  ZetkinSurveyOption,
+  ZetkinSurveyOptionsQuestionElement,
+} from 'utils/types/zetkin';
+
+export type OptionsQuestionProps = {
+  element: ZetkinSurveyOptionsQuestionElement;
+  formData: NodeJS.Dict<string | string[]>;
+};
+
+const OptionsQuestion: FC<OptionsQuestionProps> = ({ element, formData }) => {
+  const formOptions = formData[`${element.id}.options`];
+  const selectedOptions = formOptions
+    ? Array.isArray(formOptions)
+      ? formOptions
+      : typeof formOptions === 'string'
+      ? [formOptions]
+      : []
+    : [];
+
+  return (
+    <FormControl>
+      {element.question.response_config.widget_type === 'checkbox' ? (
+        <FormGroup aria-labelledby={`label-${element.id}`}>
+          <FormLabel
+            id={`label-${element.id}`}
+            style={{
+              color: 'black',
+              fontSize: '1.5em',
+              fontWeight: '500',
+              marginBottom: '0.5em',
+              marginTop: '0.5em',
+            }}
+          >
+            {element.question.question}
+          </FormLabel>
+          {element.question.options!.map((option: ZetkinSurveyOption) => (
+            <FormControlLabel
+              key={option.id}
+              control={
+                <Checkbox
+                  defaultChecked={selectedOptions.includes(`${option.id}`)}
+                  name={`${element.id}.options`}
+                />
+              }
+              label={option.text}
+              value={option.id}
+            />
+          ))}
+        </FormGroup>
+      ) : element.question.response_config.widget_type === 'radio' ? (
```

Comment:
I don't love nested ternaries. How open would you be to using the `a.b.c == 'radio && (' pattern (three times) instead?